composables/getters/productGetters.ts

```diff
@@ -65,6 +70,33 @@ export const getOptionSelectedValue = (option: ProductOption) => {
 }
 export const getOptionName = (option: ProductOption): string => option?.attributeDetail?.name || ""
 export const getOptions = (product: Product) => product?.options
+export const getSearchSuggestions = (suggestionSearch: SearchSuggestionResult) => {
```

Comment:
move getSearchSuggestions to its own getter file

Reply:
Resolved. created searchSuggestiongetters for that.